app/src/main/java/com/zulip/android/activities/MessageListFragment.java

```diff
@@ -206,11 +206,39 @@ public boolean onContextItemSelected(MenuItem item) {
                 copyMessage(message);
                 Toast.makeText(getContext(), R.string.message_copied, Toast.LENGTH_SHORT).show();
                 return true;
+            case R.id.star_message:
+                starMessage(message);
+                Toast.makeText(getContext(), R.string.message_starred, Toast.LENGTH_SHORT).show();
+                return true;
+            case R.id.un_star_message:
+                unStarMessage(message);
+                Toast.makeText(getContext(), R.string.message_un_starred, Toast.LENGTH_SHORT).show();
+                return true;
```

Comment:
I was thinking, maybe instead of toast messages we could animate the star button https://github.com/frogermcs/LikeAnimation. @kunall17 what do you think?